src/Components/Artwork/GridItem.tsx

```diff
@@ -43,6 +48,19 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
     ? transform(imageURL, { width, height })
     : { src: "", srcSet: "" }
 
+  const [linkTo, setLinkTo] = useState(artwork.href)
+
+  useEffect(() => {
+    if (!!disableRouterLinking) {
+      setLinkTo(window.location.href)
```

Comment:
I'm wondering why this has to happen in a `useEffect`? If the user has JS disabled then this will break navigation during SSR rendering. 

Also, to get the current location we should use our router directly (which is SSR compatable):

```ts
const { match } = useRouter()
...
const url = match.location.href
```

Reply:
Thank youuuuu! 🙏